app/models/qernel/reconciliation/helper.rb

```diff
@@ -28,6 +28,8 @@ def supply_demand_balance(plugin)
           .installed_adapters_of_type(:producer)
           .sum(&:carrier_demand)
 
+        #  TODO: add transformation here
```

Comment:
@noracato there's an open to do here. Should it be addressed?

Reply:
Yep